pkg/telemetry/telemetry.go

```diff
@@ -0,0 +1,166 @@
+package telemetry
+
+import (
+	"fmt"
+	"runtime"
+
+	log "github.com/charmbracelet/log"
+	cfg "github.com/cloudposse/atmos/pkg/config"
+	"github.com/cloudposse/atmos/pkg/schema"
+	"github.com/google/uuid"
+	"github.com/gruntwork-io/go-commons/version"
+	"github.com/posthog/posthog-go"
+	"github.com/spf13/cobra"
+)
+
+const (
+	DefaultTelemetryToken    = "phc_7s7MrHWxPR2if1DHHDrKBRgx7SvlaoSM59fIiQueexS"
+	DefaultTelemetryEndpoint = "https://us.i.posthog.com"
+	DefaultEventName         = "command"
+)
+
+type Telemetry struct {
+	isEnabled  bool
+	token      string
+	endpoint   string
+	distinctId string
+}
+
+func NewTelemetry(isEnabled bool, token string, endpoint string, distinctId string) *Telemetry {
+	return &Telemetry{
+		isEnabled:  isEnabled,
+		token:      token,
+		endpoint:   endpoint,
+		distinctId: distinctId,
+	}
+}
+
+func (t *Telemetry) CaptureEvent(eventName string, properties map[string]interface{}) {
+	if !t.isEnabled {
+		return
+	}
+
+	client, err := posthog.NewWithConfig(t.token, posthog.Config{
+		Endpoint: t.endpoint,
+	})
+	if err != nil {
+		log.Warn(fmt.Sprintf("Could not create PostHog client: %s", err))
+		return
+	}
+	defer client.Close()
+
+	propertiesMap := posthog.NewProperties().
+		Set("error", false).
+		Set("version", version.GetVersion()).
```

Comment:
Atmos has it's own function for getting it's version. The fact we're using `"github.com/gruntwork-io/go-commons/version"` makes me thing we're not using it.

Reply:
Done